website/docs/d/directory.html.md

```diff
@@ -23,11 +22,11 @@ data "ignition_directory" "folder" {
 
 The following arguments are supported:
 
-* `filesystem` - (Required) The internal identifier of the filesystem. This matches the last filesystem with the given identifier. This should be a valid name from a _ignition\_filesystem_ resource.
-
 * `path` - (Required) The absolute path to the directory.
 
-* `mode` - (Optional) The directory's permission mode. Note that the mode can be specified as either an octal value (e.g 0755) or a decimal value (i.e. 493 as equivalent to the octal 0755).
+* `overwrite` - (Optional) Whether to delete preexisting nodes at the path
```

Comment:
it would be great to menteion the default here: `Defaults to false.`

Reply:
will add in the next commit